### PR TITLE
fix(iOS): Add Geo (Developer Preview) to category lists.

### PR DESCRIPTION
**Pull request by atierian**
_Issue #, if available:_
N/A
_Description of changes:_
- Added Geo (Developer Preview) to various missing locations where categories / plugins are listed.
- Updated SPM screenshot to include AWSGeoLocationPlugin.
- Added Xcode minimum version pre-requisite to iOS Geo usage.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by offlineprogrammer — APPROVED** [review]

File: src/fragments/lib/project-setup/ios/create-application/20_install_libraries.mdx [hunk 1]
```diff
@@ -19,6 +19,7 @@ Lastly, choose which of the libraries you want added to your project. Always sel
 - Auth: **AWSCognitoAuthPlugin**
 - DataStore: **AWSDataStorePlugin**
 - Storage: **AWSS3StoragePlugin**
+- Geo (Developer Preview): **AWSLocationGeoPlugin**
```
Comment (ameter) [inline src/fragments/lib/project-setup/ios/create-application/20_install_libraries.mdx:22]: sort alphabetically

File: src/fragments/lib/project-setup/ios/use-existing-resources/use-existing-resources.mdx [hunk 2]
```diff
@@ -44,4 +44,5 @@ Select a category from the following list to view an example `amplifyconfigurati
 * See the [API (REST) category](/lib/restapi/existing-resources) to use existing Amazon API Gateway and AWS Lambda resources.
 * See the [Authentication category](/lib/auth/existing-resources) to use existing Amazon Cognito resources.
 * See the [Storage category](/lib/storage/existing-resources) to use existing Amazon S3 resources.
+* See the [Geo (Developer Preview) category](/lib/geo/existing-resources) to use existing Amazon Location Service resources.
```
Comment (ameter) [inline src/fragments/lib/project-setup/ios/use-existing-resources/use-existing-resources.mdx:47]: sort alphabetically

File: src/fragments/lib/project-setup/native_common/create-application/50_nextSteps.mdx [hunk 3]
```diff
@@ -7,3 +7,4 @@ Congratulations!  You've created a skeleton app and are ready to start adding Am
 * [DataStore](/lib/datastore/getting-started) - for making it easier to program for a distributed data store for offline and online scenarios
 * [Predictions](/lib/predictions/getting-started) - to detect text, images, and more!
 * [Storage](/lib/storage/getting-started) - store complex objects like pictures and videos to the cloud.
+* [Geo (Developer Preview)](/lib/geo/getting-started) - to use location data and map UI components.
```
Comment (ameter) [inline src/fragments/lib/project-setup/native_common/create-application/50_nextSteps.mdx:10]: sort alphabetically

File: src/fragments/start/getting-started/ios/nextsteps.mdx [hunk 4]
```diff
@@ -6,4 +6,5 @@
 - [API (GraphQL)](/lib/graphqlapi/getting-started)
 - [API (REST)](/lib/restapi/getting-started)
 - [Analytics](/lib/analytics/getting-started)
-- [Predictions](/lib/predictions/getting-started)
\ No newline at end of file
+- [Predictions](/lib/predictions/getting-started)
+- [Geo (Developer Preview)](/lib/geo/getting-started)
```
Comment (ameter) [inline src/fragments/start/getting-started/ios/nextsteps.mdx:10]: sort alphabetically

File: src/fragments/lib/project-setup/ios/create-application/20_install_libraries.mdx (hunk 1 above)
Comment (atierian) [reply src/fragments/lib/project-setup/ios/create-application/20_install_libraries.mdx:21]: Addressed in 4e6943f.

File: src/fragments/lib/project-setup/ios/use-existing-resources/use-existing-resources.mdx (hunk 2 above)
Comment (atierian) [reply src/fragments/lib/project-setup/ios/use-existing-resources/use-existing-resources.mdx:46]: Addressed in 4e6943f.

File: src/fragments/lib/project-setup/native_common/create-application/50_nextSteps.mdx (hunk 3 above)
Comment (atierian) [reply src/fragments/lib/project-setup/native_common/create-application/50_nextSteps.mdx]: Addressed in 4e6943f.

File: src/fragments/start/getting-started/ios/nextsteps.mdx (hunk 4 above)
Comment (atierian) [reply src/fragments/start/getting-started/ios/nextsteps.mdx]: Addressed in 4e6943f.

File: src/fragments/start/getting-started/ios/nextsteps.mdx [hunk 5]
```diff
@@ -1,9 +1,10 @@
...
-- [DataStore](/lib/datastore/getting-started)
+- [Analytics](/lib/analytics/getting-started)
 - [API (GraphQL)](/lib/graphqlapi/getting-started)
 - [API (REST)](/lib/restapi/getting-started)
-- [Analytics](/lib/analytics/getting-started)
+- [Authentication](/lib/auth/getting-started)
+- [DataStore](/lib/datastore/getting-started)
+- [Geo (Developer Preview)](/lib/geo/getting-started)
+- [Storage](/lib/storage/getting-started)
 - [Predictions](/lib/predictions/getting-started)
```
Comment (ameter) [inline src/fragments/start/getting-started/ios/nextsteps.mdx:10]:
```suggestion
- [Predictions](/lib/predictions/getting-started)
- [Storage](/lib/storage/getting-started)
```

**Review by ameter — APPROVED** [review]
Approved with suggestion.

**Pull request merged**